css/types/length.json

```diff
@@ -557,11 +558,12 @@
                 "version_added": false
               },
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/1287034'>bug 1287034</a>."
+                "version_added": "101",
+                "notes": "Starting with version 21, viewport-percentage lengths are invalid in <code>@page</code>."
```

Comment:
What does this mean? i.e. specifically, this comes in at version 101 so why a note about version 21? (everywhere this appears)?